parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -14,11 +14,9 @@ systemctlEnableAndStart() {
     RESTART_STATUS=$?
     systemctl status $1 --no-pager -l > /var/log/azure/$1-status.log
     if [ $RESTART_STATUS -ne 0 ]; then
-        echo "$1 could not be started"
         return 1
     fi
     if ! retrycmd_if_failure 120 5 25 systemctl enable $1; then
-        echo "$1 could not be enabled by systemctl"
```

Comment:
Without this logging, we won't know which `if` clause returned the 1 value. Should we use a different return value, or keep the logging?

Reply:
Granted having human messages is nicer, but I think the entire story is here:

```+ systemctlEnableAndStart rpcbind
+ systemctl_restart 100 5 30 rpcbind
+ retries=100
+ wait_sleep=5
+ timeout=30
+ svcname=rpcbind
++ seq 1 100
+ for i in '$(seq 1 $retries)'
+ timeout 30 systemctl daemon-reload
+ timeout 30 systemctl restart rpcbind
+ break
+ RESTART_STATUS=0
+ systemctl status rpcbind --no-pager -l
+ '[' 0 -ne 0 ']'
+ retrycmd_if_failure 120 5 25 systemctl enable rpcbind
+ retries=120
+ wait_sleep=5
+ timeout=25
+ shift
+ shift
+ shift
++ seq 1 120
+ for i in '$(seq 1 $retries)'
+ timeout 25 systemctl enable rpcbind
Synchronizing state of rpcbind.service with SysV init with /lib/systemd/systemd-sysv-install...
Executing /lib/systemd/systemd-sysv-install enable rpcbind
+ break
+ echo Executed '"systemctl' enable 'rpcbind"' 1 times
```

Reply:
I guess `set -x` has us covered. 👍